classes/UpgradeContainer.php

```diff
@@ -1029,6 +1033,17 @@ public function getUrlGenerator(): UrlGenerator
         return $this->urlGenerator;
     }
 
+    /**
+     * Loads classes that should normally have collisions with Prestashop dependencies.
+     */
+    public function loadNecessaryClasses(): void
+    {
+        class_exists(Table::class);
```

Comment:
Only to provide some context when we came back later on the code

```suggestion
        // Avoid collision while running on PrestaShop v9
        class_exists(Table::class);
```

Reply:
I can specify the version in the PHP doc instead, since there is already a comment

Reply:
It's only in case in the future we get more PrestaShop version with conflicts. i.e 9.1 brings more class to pre-load, then 9.2 etc...

Reply:
I hope we will have another solution in the future :)